crypto/src/transaction/builder.rs

```diff
@@ -76,41 +76,39 @@ impl Builder {
         self
     }
 
-    /// Create a new `Output` to create a new note.
-    pub fn add_output<R: RngCore + CryptoRng>(
+    /// Add a new note to the output
```

Comment:
Can the doc comment mention that this function returns the note that was added to the transaction?